scripts/tx-pull-translations.sh

```diff
@@ -48,3 +41,15 @@ for (( i=0; i<${#IOS_TRANSLATIONS[@]}; i++ )) {
 rm -rf $TMP_ANDROID_TRANSLATIONS
 
 cd ..
+
+
+
+branch=$(git rev-parse --abbrev-ref HEAD)
+if [[ ! "$branch" == *"update-transl"* ]]; then
```

Comment:
From what I understand, this will only be a little more text in the console (in case you're on a branch which doesn't have `update-transl` somewhere in its name).

Is there any documentation for the current state of translations, i.e. how to do what?

Reply:
yes, but i removed that again, as the branch name detection would already fail for the name recommended in RELEASE.md

instead, i added some lines to CONTRIBUTING.md